openmls/src/extensions/test_extensions.rs

```diff
@@ -104,18 +104,18 @@ fn ratchet_tree_extension(ciphersuite: Ciphersuite, provider: &impl OpenMlsProvi
         .merge_commit(provider, create_commit_result.staged_commit)
         .expect("error merging commit");
 
-    let bob_group = match CoreGroup::new_from_welcome(
+    let bob_group = StagedCoreWelcome::new_from_welcome(
         create_commit_result
             .welcome_option
             .expect("An unexpected error occurred."),
         None,
         bob_key_package_bundle,
         provider,
         ResumptionPskStore::new(1024),
-    ) {
-        Ok(g) => g,
-        Err(e) => panic!("Could not join group with ratchet tree extension {e}"),
-    };
+    )
+    .unwrap_or_else(|e| panic!("Could not stage group join with ratchet tree extension: {e}"))
```

Comment:
Just a nit, but `expect()` does exactly that and that's what we use elsewhere.

Reply:
`expect` does not support formating, does it?

Reply:
Oh, but it prints the error, right.